bokit/__init__.py

```
@@ -11,4 +11,4 @@
 import bokit.utils as utils
 import bokit.workflows as workflows
 
-del commands    
\ No newline at end of file
+del commands
```

Ruff (0.8.2)

Undefined name `commands`

(F821)